src/components/html/EcAdminSettingsForm.html

```diff
@@ -85,16 +100,15 @@
               component,
               data
             }) in parseAdminSettingsField(config)"
-            v-if="component"
+            v-if="component && !checkComplexSchema(schema)"
             :key="name"
-            :label="schema.title"
+            :label="header !== schema.title ? schema.title || field: ''"
```

Comment:
Considerando que só vai bater o `<a-form-item>` quando `!(checkComplexSchema(schema) && field === config.field)`, você pode deixar o label normal:

```suggestion
            :label="schema.title || field"
```

A propriedade será ignorada se o elemento for `<div>`, certo?